### ISSUE TITLE
Add screenshots to readme

### ISSUE THREAD

**Issue by matthewgiarra** [stevenlovegrove/Pangolin#724]
It's a visualization library. It would be nice to see some examples of what its visualizations actually look like. 
[The examples page](https://stevenlovegrove.github.io/Pangolin/examples/) doesn't appear to show any either. 

The main readme would be the natural place for some screen shots. Could you add a couple?

**Comment by stevenlovegrove** [comment]
You're right, the live samples I have up aren't very easy to find, a screenshot would be good.

If you follow the links on that examples page you linked, you get interactive programs compiled for web, e.g.

https://stevenlovegrove.github.io/Pangolin/emscripten/examples/SimpleDisplay/
https://stevenlovegrove.github.io/Pangolin/emscripten/examples/SimplePlot/
...
